nmostesting/suites/IS1201Test.py

```diff
@@ -110,7 +97,8 @@ def create_ncp_socket(self, test):
         self.ncp_websocket.start()
         sleep(CONFIG.WS_MESSAGE_TIMEOUT)
         if self.ncp_websocket.did_error_occur():
-            raise NMOSTestException(test.FAIL("Error opening websocket: {}".format(self.ncp_websocket.get_error_message())))
+            raise NMOSTestException(test.FAIL("Error opening websocket: {}".format(
+                self.ncp_websocket.get_error_message())))
```

Comment:
Is this change of formatting necessary?

Reply:
Are you thinking of using a `+`

Reply:
Um, I think you've just added a CR after `format(` but I might be wrong :)